### PR TITLE
Add initial versions of on-premise deploy scripts

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add initial versions of on-premise deploy scripts (*_onprem.sh)
- Update existing deploy scripts with generalized variable names

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1285

### CODE REVIEW

File: deploy_blockchain_genesis_onprem.sh [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+#        gcloud compute scp --recurse $FILES_FOR_TRACKER ${TRACKER_TARGET_ADDR}:~/ain-blockchain/ --project $PROJECT_ID --zone $TRACKER_ZONE
+#    fi
+
+    begin_index=$PARENT_NODE_INDEX_BEGIN
+    if [[ $begin_index -lt 0 ]]; then
+      begin_index=0
+    fi
+    if [[ $begin_index -le $PARENT_NODE_INDEX_END ]] && [[ $PARENT_NODE_INDEX_END -ge 0 ]]; then
+        for node_index in `seq $(( $begin_index )) $(( $PARENT_NODE_INDEX_END ))`; do
+            NODE_TARGET_ADDR="nvidia@${NODE_IP_LIST[${node_index}]}"
```
Comment (minho-comcom-ai) [inline deploy_blockchain_genesis_onprem.sh:253]: `$ONPREM_USER`

File: deploy_blockchain_genesis_onprem.sh [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+#        gcloud compute ssh $TRACKER_TARGET_ADDR --command "cd ./ain-blockchain; yarn install --ignore-engines" --project $PROJECT_ID --zone $TRACKER_ZONE
+#    fi
+
+    begin_index=$PARENT_NODE_INDEX_BEGIN
+    if [[ $begin_index -lt 0 ]]; then
+      begin_index=0
+    fi
+    if [[ $begin_index -le $PARENT_NODE_INDEX_END ]] && [[ $PARENT_NODE_INDEX_END -ge 0 ]]; then
+        for node_index in `seq $(( $begin_index )) $(( $PARENT_NODE_INDEX_END ))`; do
+            NODE_TARGET_ADDR="nvidia@${NODE_IP_LIST[${node_index}]}"
```
Comment (minho-comcom-ai) [inline deploy_blockchain_genesis_onprem.sh:303]: `$ONPREM_USER` ditto

File: deploy_blockchain_genesis_onprem.sh [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+#    eval $START_TRACKER_CMD
+#fi
+
+begin_index=$PARENT_NODE_INDEX_BEGIN
+if [[ $begin_index -lt 0 ]]; then
+    begin_index=0
+fi
+if [[ $begin_index -le $PARENT_NODE_INDEX_END ]] && [[ $PARENT_NODE_INDEX_END -ge 0 ]]; then
+    for node_index in `seq $(( $begin_index )) $(( $PARENT_NODE_INDEX_END ))`; do
+        NODE_TARGET_ADDR="nvidia@${NODE_IP_LIST[${node_index}]}"
```
Comment (minho-comcom-ai) [inline deploy_blockchain_genesis_onprem.sh:388]: `$ONPREM_USER` ditto

File: deploy_blockchain_genesis_onprem.sh (hunk 1 above)
Comment (platfowner) [reply deploy_blockchain_genesis_onprem.sh]: Done. Thanks!

File: deploy_blockchain_genesis_onprem.sh [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+#        gcloud compute ssh $TRACKER_TARGET_ADDR --command "cd ./ain-blockchain; . setup_blockchain_ubuntu_onprem.sh" --project $PROJECT_ID --zone $TRACKER_ZONE
+#    fi
+
+    begin_index=$PARENT_NODE_INDEX_BEGIN
+    if [[ $begin_index -lt 0 ]]; then
+      begin_index=0
+    fi
+    if [[ $begin_index -le $PARENT_NODE_INDEX_END ]] && [[ $PARENT_NODE_INDEX_END -ge 0 ]]; then
+        for node_index in `seq $(( $begin_index )) $(( $PARENT_NODE_INDEX_END ))`; do
+            NODE_TARGET_ADDR="nvidia@${NODE_IP_LIST[${node_index}]}"
```
[the remark this answers is not in the document]
Comment (platfowner) [reply deploy_blockchain_genesis_onprem.sh]: Done. Thanks!

File: deploy_blockchain_genesis_onprem.sh (hunk 2 above)
Comment (platfowner) [reply deploy_blockchain_genesis_onprem.sh]: Done. Thanks!

File: deploy_blockchain_genesis_onprem.sh [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+#        gcloud compute ssh $TRACKER_TARGET_ADDR --command "sudo killall node" --project $PROJECT_ID --zone $TRACKER_ZONE
+#    fi
+
+    begin_index=$PARENT_NODE_INDEX_BEGIN
+    if [[ $begin_index -lt 0 ]]; then
+      begin_index=0
+    fi
+    if [[ $begin_index -le $PARENT_NODE_INDEX_END ]] && [[ $PARENT_NODE_INDEX_END -ge 0 ]]; then
+        for node_index in `seq $(( $begin_index )) $(( $PARENT_NODE_INDEX_END ))`; do
+            NODE_TARGET_ADDR="nvidia@${NODE_IP_LIST[${node_index}]}"
```
[the remark this answers is not in the document]
Comment (platfowner) [reply deploy_blockchain_genesis_onprem.sh]: Done. Thanks!

File: deploy_blockchain_genesis_onprem.sh (hunk 3 above)
Comment (platfowner) [reply deploy_blockchain_genesis_onprem.sh]: Done. Thanks!

**Review by jiyoung-an — APPROVED** [review]
LGTM, Thanks!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an @minho-comcom-ai Now merging.. and let me handle additional comments in the coming prs if any.

**Pull request merged**